### ISSUE TITLE
Question about a step in README

### ISSUE THREAD

**Issue by NotOneRing** [irom-princeton/dppo#21]
Hi Allen,
This is the explanation in README.md
3. Install specific environment dependencies (Gym / Kitchen / Robomimic / D3IL / Furniture-Bench) or all dependencies (except for Kitchen, which has dependency conflicts with other tasks).
pip install -e .[gym] # or [kitchen], [robomimic], [d3il], [furniture]
pip install -e .[all] # except for Kitchen

After the execution:

pip install -e .[gym] 
zsh: no matches found: .[gym]

May I ask what should be done for the next step?

**Comment by allenzren** [comment]
I think it has to with zsh https://stackoverflow.com/questions/30539798/zsh-no-matches-found-requestssecurity. Maybe try `pip install -e '.[gym]'`

**Comment by NotOneRing** [comment]
Hi Allen,

Thanks so much for your reply!!!

May I ask what could be the minimum GPU requirements to conduct experiments? I noticed that you used one Nvidia L40 in your paper.

**Comment by allenzren** [comment]
Hey! Which experiment are you interested in? If it is just state input then you don't need many VRAM. With image input, you can use [gradient accumulation](https://github.com/irom-princeton/dppo/blob/c0921a1fb5cb8f8559a5e5ba3ca87eacaa19f390/cfg/robomimic/finetune/lift/ft_ppo_diffusion_mlp_img.yaml#L68) to reduce VRAM requirement.

**Comment by NotOneRing** [comment]
Hi Allen,

Thank you so much for your reply!!! I will have a try!
